### PR TITLE
Initialize repository

### PR DESCRIPTION
**Pull request by muvaf**
Usual ritual.

One thing I need to do some extra work is whether it should be `provider-influxdb-cloud` or just `provider-influxdb` because it seems like InfluxDB Cloud provides the same API as OSS. For example, you can [create organizations](https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/org/create/#works-with-influxdb-oss-2x) in both but cloud requires a new sign-up whereas you can just call the API and there is only a [single Go](https://docs.influxdata.com/influxdb/cloud/api-guide/client-libraries/go/) client targeting both.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,42 +1,90 @@
...
+	@make
+
+# Generate a coverage report for cobertura applying exclusions on
+# - generated file
+cobertura:
+	@cat $(GO_TEST_OUTPUT)/coverage.txt | \
+		grep -v zz_ | \
+		$(GOCOVER_COBERTURA) > $(GO_TEST_OUTPUT)/cobertura-coverage.xml
+
+crds.clean:
```
Comment (turkenh) [inline Makefile:71]: Shouldn't this be invoked somewhere? e.g.
```
generate.done: crds.clean
```

File: apis/template.go [hunk 2]
```diff
@@ -20,14 +20,14 @@ package apis
 import (
```
Comment (turkenh) [inline apis/template.go:20]: filename needs to be changed as well.

File: apis/v1alpha1/groupversion_info.go [hunk 3]
```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 // Package v1alpha1 contains the core resources of the Template provider.
```
Comment (turkenh) [inline apis/v1alpha1/groupversion_info.go:17]:
```suggestion
// Package v1alpha1 contains the core resources of the InfluxDB provider.
```

File: package/crds/influxdb.crossplane.io_providerconfigs.yaml [hunk 4]
```diff
@@ -1,12 +1,14 @@
+
```
Comment (turkenh) [inline package/crds/influxdb.crossplane.io_providerconfigs.yaml:1]: will be removed with `generate.done: crds.clean` to makefile

**Review by turkenh — APPROVED** [review]

**Comment by muvaf** [comment]
@turkenh I addressed your comments here https://github.com/crossplane-contrib/provider-influxdb/pull/2

**Pull request merged**